.gitignore

```diff
@@ -1,2 +1,4 @@
 danavis/
-.vscode/
\ No newline at end of file
+.vscode/
+*.json
+.DS*
```

Comment:
the star here is a bit odd.

Reply:
Could exclude a large group of unintended files.

Reply:
ps. th .DS_store is still on the github. Challenge: remove it ;)

Reply:
Haha, you bastard! Thought I'd have to wrangle a lot with git here, but it's done ;-)